projects/Time to load Website/time_to_load_website.py

```diff
@@ -0,0 +1,27 @@
+from urllib.request import urlopen
+import time
+
+#Storing user defined URl
+URL = input("Enter the url whose loading time you want to check: ")
```

Comment:
name 'URL' should be lowercase(Constant should be defined in Upper case)
```suggestion
url = input("Enter the url whose loading time you want to check: ")
```